src/main/java/org/prgrms/application/CommandType.java

```diff
@@ -0,0 +1,17 @@
+package org.prgrms.application;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum CommandType {
+    EXIT,
+    CREATE,
+    LIST;
+
+
+    public static Optional<CommandType> findBySelection(String selection) {
+        return Arrays.stream(values())
+                .filter(s -> s.name().equals(selection.toUpperCase()))
+                .findFirst();
+    }
```

Comment:
p1;
```suggestion
public static Optional<CommandType> findBySelection(String selection) {
        return Arrays.stream(values())
                .filter(s -> s.name().equals(selection.toUpperCase()))
                .findFirst()
                .orElseThrow(() -> new RuntimeException("잘못된 입력입니다."));
    }
```
위 코드처럼 CommandType에서 예외 책임을 가져가보기!